components/app/Calendar.vue

```diff
@@ -93,7 +93,84 @@ const onEventClick = (event: any, e: any) => {
           List
         </label>
       </div>
-    </div> -->
+    </div>
+    <vue-cal
+      v-if="selectedView === 'calendar'"
+      class="rounded-lg bg-white dark:bg-neutral-900 overflow-hidden shadow border border-gray-400/40"
+      today-button
+      small
+      :events-on-month-view="true"
+      :twelve-hour="true"
+      :events="group.items"
+      :start-week-on-sunday="true"
+      :disable-views="['years', 'year', 'day']"
+      :time-from="8 * 60"
+      :time-to="22 * 60"
+      :time-step="30"
+      :on-event-click="onEventClick"
+      :show-all-day-events="['short', true, false]"
+    >
+      <template #arrow-prev>
+        <Icon
+          class="w-8 h-8"
+          name="i-ph-arrow-left"
+        />
+      </template>
+      <template #arrow-next>
+        <Icon
+          class="w-8 h-8"
+          name="i-ph-arrow-right"
+        />
+      </template>
+    </vue-cal>
+
+    <!-- List View -->
+    <div
+      v-if="selectedView === 'list'"
+    >
+      <div
+        v-for="(events, month) in groupedEvents"
+        :key="month"
+        class="mb-8"
+      >
+        <h3 class="text-2xl text-center mb-8 font-bold tracking-tight text-gray-900 dark:text-white">
+          {{ month }}
+        </h3>
+        <div class="grid xl:grid-cols-2 gap-4">
+          <div
+            v-for="(event) in events"
+            :key="event.title"
+            class="border-2 border-gray-400/40 rounded mb-4 break-word"
+          >
+            <div class="p-4 flex justify-between items-center border-b rounded-t border-gray-400/40">
+              <h3 class="text-xl font-bold tracking-tight text-gray-900 dark:text-white">
+                {{ event.title }}
+              </h3>
+              <div class="flex flex-col items-end gap-1">
+                <div class="flex items-center gap-4">
+                  <Icon name="formkit:date" />
+                  <p class="text-sm text-gray-500">
+                    {{ event.start.format('DD/MM/YYYY') }}
+                  </p>
+                </div>
+                <div class="flex items-center gap-2">
+                  <Icon name="mingcute:time-line" />
+                  <p class="text-sm text-gray-500">
+                    {{ event.start.formatTime() }} - {{ event.end.formatTime() }}
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="p-4 event-description">
```

Comment:
@tylersocholotuik sometimes the events are going to have a bit of a longer description because of markdown from.

This technique I wouldn't use all the time but I think this will solve our problem and look good.

```suggestion
            <div
              class="p-4 event-description"
              :class="{ 'max-w-sm': isMobile }"
            >
```

The mobile view on pixel 7:
![image](https://github.com/user-attachments/assets/708f3a73-d11e-4374-9a79-b5961ae136b1)

The desktop view:
![image](https://github.com/user-attachments/assets/17d7ee13-2d9c-4d7a-adfe-828b578e7d8c)

@arashsheyda @tylersocholotuik let me know what you think.

I'd be ready to merge this once this implemented so that it doesn't break on mobile formats.